pages/desktop/developer_hub/content_tools.py

```diff
@@ -0,0 +1,109 @@
+#  This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.support.ui import WebDriverWait
+
+from pages.desktop.developer_hub.base import Base
+from pages.page import Page, PageRegion
+
+
+class ContentTools(Base):
+
+    _submit_new_add_on_button_locator = (By.CSS_SELECTOR, '.landing--submit a')
+    _agree_button_locator = (By.CSS_SELECTOR, '.sign button')
+    _submit_add_on_form_button_locator = (By.CSS_SELECTOR, '.addon-upload button[type="submit"]')
+    _select_add_on_file_button_locator = (By.CLASS_NAME, 'form-inline--file-input')
+    _add_on_file_input_locator = (By.ID, 'submission-addon--zip')
+    _notification_message_locator = (By.CSS_SELECTOR, '.notification')
+    _add_on_locator = (By.CSS_SELECTOR, 'li.addon-for-listing')
+
+    @property
+    def header(self):
+        return self.HeaderRegion(self.testsetup)
+
+    @property
+    def notification_message(self):
+        return self.selenium.find_element(*self._notification_message_locator).text
+
+    @property
+    def add_ons(self):
+        return [self.AddOn(self.testsetup, web_element)
+                for web_element in self.selenium.find_elements(*self._add_on_locator)]
+
+    def add_on(self, name):
+        for add_on in self.add_ons:
+            if add_on.name == name:
+                return add_on
```

Comment:
I think we could use [next](https://docs.python.org/2/library/functions.html#next) here, such as:

```python
def add_on(self, name):
    next(a for a in self.add_ons if a.name == name)
```